list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+$query = "SELECT * FROM books";
+$result = mysqli_query($connection, $query);
+?>
+<!DOCTYPE html>
+<html>
```

Comment:
Observations: Error handling is done by echoing database errors directly to the user, which is not user-friendly and can expose sensitive information.
Recommendations: Implement a logging mechanism to log errors and display a generic error message to the user.